sql/core/src/test/resources/tpcds-plan-stability/approved-plans-modified/q27.sf100/explain.txt

```diff
@@ -128,15 +128,15 @@ Input [9]: [ss_item_sk#1, ss_cdemo_sk#2, ss_store_sk#3, ss_quantity#4, ss_list_p
 Output [4]: [cd_demo_sk#13, cd_gender#14, cd_marital_status#15, cd_education_status#16]
 Batched: true
 Location [not included in comparison]/{warehouse_dir}/customer_demographics]
-PushedFilters: [IsNotNull(cd_gender), IsNotNull(cd_marital_status), IsNotNull(cd_education_status), EqualTo(cd_gender,F), EqualTo(cd_marital_status,D), EqualTo(cd_education_status,Primary), IsNotNull(cd_demo_sk)]
+PushedFilters: [IsNotNull(cd_gender), IsNotNull(cd_marital_status), IsNotNull(cd_education_status), EqualTo(cd_gender,F), EqualTo(cd_marital_status,D), EqualTo(cd_education_status,Primary             ), IsNotNull(cd_demo_sk)]
```

Comment:
No big deal at all, just wondering why this now has whitespace? is it because these are now fixed-length strings?

Reply:
`cd_education_status` is char type, which means the data of it in the storage are always padded as fixed-length strings. While before it's string type and doesn't have this issue.

Reply:
the `dsdgen` tool would produce fixed-length strings for char values in the original data, so we should pad accordingly to match correctlly